go-controller/pkg/metrics/master.go

```diff
@@ -23,14 +23,27 @@ import (
 	klog "k8s.io/klog/v2"
 )
 
-// metricE2ETimestamp is a timestamp value we have persisted to nbdb. We will
-// also export a metric with the same column in sbdb. We will also bump this
-// every 30 seconds, so we can detect a hung northd.
-var metricE2ETimestamp = prometheus.NewGauge(prometheus.GaugeOpts{
+// metricNbE2eTimestamp is the UNIX timestamp value set to NB DB. A corresponding metric
+// 'sb_e2e_timestamp' for SB DB will contain the timestamp that was written to NB DB.
```

Comment:
I am reading this right and the comment line should end with SB DB?

Reply:
Its correct depending on which way you look at it. Ill make it clearer. 
What happens is ovnkube master writes the current timestamp to NB DB. Northd copies that to SB DB and we read it and store it in metric `sb_e2e_timestamp`.